### PR TITLE
feat: Implement CreateLink and setup linker test in go-service-fs

### PR DESCRIPTION
**Pull request by abyss-w**
feat: Implement CreateLink and setup linker test in go-service-fs

### CODE REVIEW

File: storage.go [hunk 1]
```diff
@@ -287,7 +336,9 @@ func (s *Storage) stat(ctx context.Context, path string, opt pairStorageStat) (o
 	if fi.Mode()&os.ModeSymlink != 0 {
 		o.Mode |= ModeLink
 
-		target, err := filepath.EvalSymlinks(rp)
+		//target, err := filepath.EvalSymlinks(rp)
```
Comment (Xuanwo) [inline storage.go:339]: Please remove not-used comments.

File: storage.go [hunk 2]
```diff
@@ -286,8 +335,9 @@ func (s *Storage) stat(ctx context.Context, path string, opt pairStorageStat) (o
 	// Check if this file is a link.
 	if fi.Mode()&os.ModeSymlink != 0 {
 		o.Mode |= ModeLink
-
-		target, err := filepath.EvalSymlinks(rp)
+		
+		// FIXME
```
Comment (Xuanwo) [inline storage.go:339]: Please give more information for `FIXME`.

- What's the problem?
- What we did for this problem?

Comment (Xuanwo) [reply storage.go:339]: Does it related to https://github.com/beyondstorage/go-service-fs/pull/64?

Comment (abyss-w) [reply storage.go:339]: > Is it related to #64?

Yes, it is.

Comment (Xuanwo) [reply storage.go:339]: > > Is it related to #64?
> 
> Yes, it is.

This is not good.

Please don't assume that others know the context, and do your best to put all the information you know in a comment or at least in a PR.

Imagine if you were an external contributor and saw a line like this FIXME, how would you feel? Even if you wanted to contribute, you wouldn't be able to participate.

The most important thing for BeyondStorage: **Be open**.

Comment (abyss-w) [reply storage.go:339]: Got it, I will take care of that in the future.

**Review by Xuanwo — APPROVED** [review]

**Pull request merged**